### ISSUE TITLE
Muscat, authorities, new index

### ISSUE THREAD

**Issue by Docudoctor** [rism-digital/muscat#1381]
Please create a new index called "Text incipits", entries taken from (031) in source records.

**Comment by Docudoctor** [comment]
See also #1347

**Comment by Docudoctor** [comment]
We need the four standard actions "Create" and "Merge", also "Add all to a new folder" and "Add all to an existing folder"
Not to forget the magnifying glass in 031 of sources.

**Comment by xhero** [comment]
So unfortunately with how we structure our data none of those actions are really possible to implement.
What are your exact requirements to do? I imagine it would be like for the standard titles, where changing an entry in the index automatically fixes the sources?

**Comment by Docudoctor** [comment]
Yes. Same actions/ functions like for the standard titles, especially Create and Merge.
